util.hpp

```diff
@@ -21,7 +21,7 @@ namespace Sass {
   string comment_to_string(const string& text);
   string normalize_wspace(const string& str);
 
-  string quote(const string&, char q = 0);
+  string quote(const string&, char q = 0, bool flag = false);
```

Comment:
What does `flag` mean? Is there a better name that will describe what this is for?